compiler/testData/codegen/box/companion/kt14258.kt

```diff
@@ -0,0 +1,12 @@
+// Checks that methods 'access$getMy$p', 'access$getMy$cp' and 'getMy' are not generated and
```

Comment:
I think this box test is unnecessary since it doesn't really check what's described here, it only checks that private properties in companion objects work, which is done elsewhere, so please delete this test

Reply:
Remove, you are right, the runtime behavior is already check by others tests.